src/docfx/cli/Docfx.cs

```diff
@@ -129,6 +129,10 @@ private static (string command, string workingDirectory, CommandLineOptions opti
                     syntax.DefineOption("no-dry-sync", ref options.NoDrySync, "Do not run dry sync for learn validation.");
                     syntax.DefineOption("no-restore", ref options.NoRestore, "Do not restore dependencies before build.");
                     syntax.DefineOption("no-cache", ref options.NoCache, "Do not use cache dependencies in build, always fetch latest dependencies.");
+                    syntax.DefineOption(
```

Comment:
move to `DefineCommonOptions` because `--template` is also defined there?

Reply:
but `--template-base-path` can be used separately, and it is not useful for other commands except `build`.

Reply:
It is not useful for `restore` but is useful for other `build` like commands e.g., `serve` (which we currently don't have). I might be overthinking, we could figure it out later.

Reply:
OK, but it still not a common option for all the command~😃